lib/MusicBrainz/Server/Data/WikipediaExtract.pm

```diff
@@ -175,7 +175,8 @@ sub _extract_by_language_callback
     my (%opts) = @_;
     if ($opts{fetched}{content}) {
         return WikipediaExtract->new( title => $opts{fetched}{title},
-                                      content => $opts{fetched}{content},
+                                      content => sprintf '<bdi>%s</bdi>',
```

Comment:
I suspect this will also cause HTML errors because the content is likely to have HTML not allowed inside `<bdi>`.